modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-web/src/main/java/com/liferay/dynamic/data/mapping/form/web/internal/display/context/util/DDMFormInstanceSubmissionLimitStatusUtil.java

```diff
@@ -69,14 +69,10 @@ public static boolean isSubmissionLimitReached(
 		Optional<DDMFormInstanceRecordVersion> optional = stream.filter(
 			ddmFormInstanceRecordVersion ->
 				ddmFormInstanceRecordVersion.getStatus() !=
-					WorkflowConstants.STATUS_DRAFT
+					WorkflowConstants.ACTION_PUBLISH
```

Comment:
@Gabriel422-hue @SamZiemer we must keep the `WorkflowConstants.STATUS_DRAFT` otherwise it will change the expected behavior..

Reply:
Thank you @carolmariaabb, Do you have any suggestions on how to resolve the issue in that case? With `WorkflowConstants.STATUS_DRAFT` resubmissions are not currently allowed after a rejection when single submission is selected.

Reply:
Will it work if you do: `ddmFormInstanceRecordVersion.getStatus() != WorkflowConstants.STATUS_DRAFT && ddmFormInstanceRecordVersion.getStatus() != WorkflowConstants.STATUS_PENDING`?

Reply:
@carolmariaabb I'm ll' check and talk to you! Thanks!